src/morphology.cpp

```diff
@@ -47,6 +47,9 @@ Morphology::Morphology(const Property::Properties& properties, unsigned int opti
 Morphology::Morphology(const HighFive::Group& group, unsigned int options)
     : Morphology(readers::h5::load(group), options) {}
 
+Morphology::Morphology(const std::stringstream& stream, unsigned int options)
+    : Morphology(readers::asc::load(stream.str(), "Stream", options), options) {}
```

Comment:
This new ctor assume the stream is in NeuroLucida format.
That's probably not good enough. Maybe we should be able to pass the format as a parameter ?